backend/ops_api/ops/utils/cans.py

```diff
@@ -87,15 +108,109 @@ def get_can_funding_summary(can: CAN, fiscal_year: Optional[int] = None) -> CanF
     available_funding = total_funding - sum([planned_funding, obligated_funding, in_execution_funding]) or 0
 
     return {
-        "can": can.to_dict(),
```

Comment:
Would be better as a Marshmallow schema for the response but OK as tech debt.

Reply:
I added the items below to avoid tech debt: 
- request and response Marshmallow schemas
- new service for can funding summary